ui/src/app/shared/workflow/node/context/workflow.node.context.component.ts

```diff
@@ -60,6 +70,13 @@ export class WorkflowNodeContextComponent {
             this._variableService.getContextVariable(this.project.key, this.node.pipeline_id)
               .subscribe((suggest) => this.suggest = suggest);
 
+            if (this.node.context && this.node.context.application) {
```

Comment:
&& this.node.context.application.repoxxxxx  ?  if not api will search in pipelinebuild history :)